test/certs/acert.pem

```diff
@@ -0,0 +1,15 @@
+-----BEGIN ATTRIBUTE CERTIFICATE-----
+MIICPTCCASUCAQEwN6AWMBGkDzANMQswCQYDVQQDDAJDQQIBAqEdpBswGTEXMBUG
+A1UEAwwOc2VydmVyLmV4YW1wbGWgLTArpCkwJzElMCMGA1UEAwwcQXR0cmlidXRl
+IENlcnRpZmljYXRlIElzc3VlcjANBgkqhkiG9w0BAQsFAAIUA7WQWQKiqrVAIUS4
+LE/ZgBtfV8IwIhgPMjAyMTA2MTUxMjM1MDBaGA8yMDMxMDYxMzEyMzUwMFowQTAj
+BggrBgEFBQcKBDEXMBWgCYYHVGVzdHZhbDAIDAZncm91cDEwGgYDVQRIMRMwEaEP
+gw1hZG1pbmlzdHJhdG9yMCwwHwYDVR0jBBgwFoAUYm7JaGdsZLtTgt0tqoCK2MrI
+i10wCQYDVR04BAIFADANBgkqhkiG9w0BAQsFAAOCAQEARYpFEjordN68NXSLmDae
+uruxhw+Zsr9Grom4mqYy/2lAuu58os4xA4Cez9OdkfCZmU9baDBVfMWys7GlCJdg
+wn5uO5Kk2eiEWsujO/EV4c5eyXSrsUghQZQKfIVsgeut+7WfmDy/+j8ibbxJxJp/
+KMIoDjBAaMDhhxaCYclm8dJRT9DPba/bCNjuPGeTXslDQSXIfzFDgQkRZ0+Z7s9i
+enYBmnH4rvW+SX8v5MzfGOu8VtHAKKBEcCbs2qGwYBEUUsCA/LYKSnOwn655wsd9
+k6KifIGOCVS4ZbK9pDyYcH/6/OmD1eSUwDZsd8CrJYDC31gAu7xqhOsBzR9DL3oD
+4Q==
+-----END ATTRIBUTE CERTIFICATE-----
```

Comment:
What is the providence of this test certificate? Are there existing implementations whose output, or own test files, we can add to the test suite?

Reply:
This was created by me using bouncycastle. I will check again to see if I can find any test files from other implementations.

Reply:
Is there any advice on how to handle the licensing of PEM file test data?
For example, I have found a test certificate in the bouncycastle source, licensed under MIT.  Is it sufficient to add something like the following header to the PEM file?

```
Extracted from bouncycastle test case
Source: pkix/src/test/j2me/org/bouncycastle/cert/test/AttrCertTest.java
Copyright (c) 2000-2023 The Legion of the Bouncy Castle Inc. (https://www.bouncycastle.org)
SPDX-License-Identifier: MIT

-----BEGIN ATTRIBUTE CERTIFICATE-----
...
```

Reply:
This was always a problem. Unfortunately the CLA has to be applicable to the test data as well so IMO you cannot use this. Of course you can use certs you generated yourself by some 3rd party software.

Reply:
Ah yes, of course.  

NSA Cyber has a tool to generate platform certificates (https://github.com/nsacyber/paccor).  I'll try to generate something with that.

Reply:
Hi, we've been sent a copy of the CLA and I'm in the process of signing it. You're welcome to make use of any test data we have published in the meanwhile (and anything else for that matter).

David Hook
Secretary
Legion of the Bouncy Castle Inc.
office@bouncycastle.org

Reply:
Oh, that's amazing.  Thank you very much.

@t8m, assuming that there is a CLA for bouncycastle on file, is the above header ok?  Should I add an "Acked-by" or something else to the commit message?  In that case, I think it makes sense to split this test data into a separate patch.

Reply:
It was just emailed to the legal address.

Reply:
We have received it - it will be processed shortly.
Thanks @dghgit for sorting this out as it really helps to have good test data.

Reply:
I've added two certificates from bouncycastle test cases to the MR as a separate patch.
It seems to have been flagged, pending the CLA processing.

Reply:
> It seems to have been flagged, pending the CLA processing.

The cla bot is complaining because you seem to have used two different author email addresses (both of them seem to be you) - but only one of them is in the CLA database.

Reply:
Thank you. I'll fix that, and I noticed that I have some CI failures too, so I will address those as well and re-push.

Reply:
@dhobsong 
I made some platform certificates (attribute certificates). I am happy if it helps.
please find the attach file, [acerts.zip](https://github.com/openssl/openssl/files/14414767/acerts.zip)

If there is a proper method of contribution other than attaching files, I would appreciate your guidance.

Reply:
@tatac1 Thank you for the certificates.  

If you want to contribute these, I think that you will also need to submit a [CLA](https://www.openssl.org/policies/cla.html), as it says in the CONTRIBUTING.md file.

This MR now already includes one TPM platform certificate, with similar fields to the certificates you provided, so that may be enough for now.  Your certificates use more attribute types, so should be very useful in future as support for more types of attributes are added.

Reply:
> @tatac1 Thank you for the certificates.
> 
> If you want to contribute these, I think that you will also need to submit a [CLA](https://www.openssl.org/policies/cla.html), as it says in the CONTRIBUTING.md file.
> 
> This MR now already includes one TPM platform certificate, with similar fields to the certificates you provided, so that may be enough for now. Your certificates use more attribute types, so should be very useful in future as support for more types of attributes are added.

I have signed an Individual Contributor License Agreement and sent it.

Reply:
> I have signed an Individual Contributor License Agreement and sent it.

Good to know that there are more certificates available if needed.